### PR TITLE
Update GCP credentials docs

### PR DESCRIPTION
**Pull request by GGP1**
## Description

Improves the GCP credentials documentation by adding a new section explaining how to create a custom role for the gcp-bucket module and a service account creation image.

### Docs building
- [x] Compiles without warnings.
### Code formatting and web optimization
- [x] Uses three spaces indentation.
- [x] Adds or updates meta descriptions accordingly.
- [x] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).
### Writing style
- [x] Uses present tense, active voice, and semi-formal registry.
- [x] Uses short, simple sentences.
- [x] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.

### CODE REVIEW

File: source/cloud-security/gcp/prerequisites/credentials.rst [hunk 1]
```diff
@@ -13,14 +13,37 @@ In order to make the Wazuh GCP module pull log data from Google Pub/Sub or Googl
...
+   - ``storage.bucket.get``
+   - ``storage.objects.create``
+   - ``storage.objects.delete``
+   - ``storage.objects.list``
+
+And click on **Create**.
+
+.. thumbnail:: /images/cloud-security/gcp/gcp-bucket-role.png
+    :align: center
+    :width: 100%
```
Comment (fdalmaup) [inline source/cloud-security/gcp/prerequisites/credentials.rst:34]:
```suggestion
To do so, follow these steps:

1. Go to the **Roles** section and click on **Create Role**.
2. Once you have chosen a name, click on **Add Permissions**.
3. On the search bar type **Storage Legacy Bucket Writer** and assign the following permissions:

   - ``storage.bucket.get``
   - ``storage.objects.create``
   - ``storage.objects.delete``
   - ``storage.objects.list``

.. thumbnail:: /images/cloud-security/gcp/gcp-bucket-role.png
    :align: center
    :width: 100%

4. Click on **Create**.
```

Also, the different fields could be highlighted in the image.


```

**Review by fdalmaup — CHANGES_REQUESTED** [review]

File: source/cloud-security/gcp/prerequisites/credentials.rst [hunk 2]
```diff
@@ -13,14 +13,39 @@ In order to make the Wazuh GCP module pull log data from Google Pub/Sub or Googl
...
+1. Go to the **Roles** section and click on **Create Role**.
+2. Choose a name and click on **Add Permissions**.
+3. On the search bar, type **Storage Legacy Bucket Writer** and select the following permissions:
+
+   - ``storage.bucket.get``
+   - ``storage.objects.create``
+   - ``storage.objects.delete``
+   - ``storage.objects.list``
+
+4. Click on **Create**.
```
Comment (GGP1) [inline source/cloud-security/gcp/prerequisites/credentials.rst:32]: In the previous suggestion the 4th point was below the image but it didn't look so good once compiled, so I included this point below the other ones

screenshot

**Review by fdalmaup — APPROVED** [review]
LGTM!

**Review by Selutario — APPROVED** [review]

File: source/cloud-security/gcp/prerequisites/credentials.rst [hunk 3]
```diff
@@ -13,14 +13,39 @@ In order to make the Wazuh GCP module pull log data from Google Pub/Sub or Googl
 
 To do this, it is recommended to create a service account with the Pub/Sub or Storage permissions and then create a key. It is important to save this key as a JSON file as it will be used as the authentication method for the GCP module.
 
+Creating a custom role
+----------------------
+
+The ``gcp-bucket`` module requires permissions to access storage buckets and objects. These permissions were originally assigned using the **Storage Legacy Bucket Writer** role, but it is no longer available for selection, so it will be necessary to create a new one.
```
Comment (javimed) [inline source/cloud-security/gcp/prerequisites/credentials.rst:19]: Originally... but it is no longer available..? When? What does all this mean?

File: source/cloud-security/gcp/prerequisites/credentials.rst [hunk 4]
```diff
@@ -13,14 +13,39 @@ In order to make the Wazuh GCP module pull log data from Google Pub/Sub or Googl
...
+
+To do so, follow these steps:
```
Comment (javimed) [inline source/cloud-security/gcp/prerequisites/credentials.rst:21]:
```suggestion
To create the new role, follow these steps:
```

File: source/cloud-security/gcp/prerequisites/credentials.rst [hunk 5]
```diff
@@ -13,14 +13,39 @@ In order to make the Wazuh GCP module pull log data from Google Pub/Sub or Googl
...
+
+1. Go to the **Roles** section and click on **Create Role**.
```
Comment (javimed) [inline source/cloud-security/gcp/prerequisites/credentials.rst:23]:
```suggestion
#. Go to the **Roles** section and click on **Create Role**.
```

File: source/cloud-security/gcp/prerequisites/credentials.rst [hunk 6]
```diff
@@ -13,14 +13,39 @@ In order to make the Wazuh GCP module pull log data from Google Pub/Sub or Googl
...
+2. Choose a name and click on **Add Permissions**.
```
Comment (javimed) [inline source/cloud-security/gcp/prerequisites/credentials.rst:24]:
```suggestion
#. Establish a **Title** and click on **Add Permissions**.
```

File: source/cloud-security/gcp/prerequisites/credentials.rst [hunk 7]
```diff
@@ -13,14 +13,39 @@ In order to make the Wazuh GCP module pull log data from Google Pub/Sub or Googl
...
+3. On the search bar, type **Storage Legacy Bucket Writer** and select the following permissions:
```
Comment (javimed) [inline source/cloud-security/gcp/prerequisites/credentials.rst:25]:
```suggestion
#. On the search bar, filter available permissions by typing **Storage Legacy Bucket Writer**. Select the following ones:
```

**Review by javimed — CHANGES_REQUESTED** [review]

File: source/cloud-security/gcp/prerequisites/credentials.rst (hunk 3 above)
Comment (javimed) [inline source/cloud-security/gcp/prerequisites/credentials.rst:19]:
```suggestion
The ``gcp-bucket`` module requires permissions to access storage buckets and objects. To create a role with the required permissions, follow these steps:
```

File: source/cloud-security/gcp/prerequisites/credentials.rst [hunk 8]
```diff
@@ -13,14 +13,39 @@ In order to make the Wazuh GCP module pull log data from Google Pub/Sub or Googl
 
 To do this, it is recommended to create a service account with the Pub/Sub or Storage permissions and then create a key. It is important to save this key as a JSON file as it will be used as the authentication method for the GCP module.
 
+Creating a custom role
+----------------------
+
+The ``gcp-bucket`` module requires permissions to access storage buckets and objects. These permissions were originally assigned using the **Storage Legacy Bucket Writer** role, but it is no longer available for selection, so it will be necessary to create a new one.
+
+To create the new role, follow these steps:
```
Comment (javimed) [inline source/cloud-security/gcp/prerequisites/credentials.rst:21]:
```suggestion
```

File: source/cloud-security/gcp/prerequisites/credentials.rst [hunk 9]
```diff
@@ -13,14 +13,39 @@ In order to make the Wazuh GCP module pull log data from Google Pub/Sub or Googl
...
+
```
Comment (javimed) [inline source/cloud-security/gcp/prerequisites/credentials.rst:22]:
```suggestion
```

**Review by javimed — CHANGES_REQUESTED** [review]

**Review by javimed — APPROVED** [review]

**Pull request merged**